friends/src/components/PrivateRoute.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import { Route, Redirect } from "react-router-dom";
+
+const PrivateRoute = ({ component: Component, ...theRest}) => {
+    return(
+        <Route
+        {...theRest}
+        render = {() => {
+            if (localStorage.getItem("token")){
```

Comment:
I have a question why did you call it theRest? I am curious because sometimes I create naming conventions that make sense to me. But maybe not other people so this was just something I was curious about.